src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -495,6 +495,26 @@ public object PrivateData
         [ValidateNotNullOrEmpty]
         public string ReleaseNotes { get; set; }
 
+        /// <summary>
+        /// Specify that the module is prerelease.
+        /// </summary>
+        [Parameter(Mandatory = false)]
```

Comment:
No need to set defaults.
```suggestion
        [Parameter]
```
Below too.